### ISSUE TITLE
BGC parameters

### ISSUE THREAD

**Issue by aekiss** [hakaseh/input_om2-bgc#3]
Hi @hakaseh, I have some questions below about some of the parameter values
```python
param_list = [
 ['abio',3.125e-6],
 ['alphabio',2.96296e-6],
 ['bbio',1.066],
 ['cbio',1],
 ['epsbio',1.85185e-5],
 ['fe_bkgnd',0.6],
 ['f_inorg',0.062], # where is this from? not in Ziehn (2020) or Hayashida (2020) tables
 ['gam1bio',0.925],
 ['gam2bio',1.15741e-7],
 ['gbio',1.82292e-5],
 ['kcbio',21.2766], # where is this from? not in Ziehn (2020) or Hayashida (2020) tables
 ['kwbio',25], # where is this from? not in Ziehn (2020) or Hayashida (2020) tables
 ['k1bio',0.7],
 ['muecaco3',1.9838e-8], # where is this from? not in Ziehn (2020) or Hayashida (2020) tables
 ['muecaco3_sed',4.05093e-8],
 ['muedbio',5.55556e-7],
 ['muedbio_sed',2.31481e-7], # doesn't match Ziehn (2020) 0.024/(24*60*60)=2.77778e-07 or Hayashida (2020) table, which has muedbio_sed=muedbio=5.55556e-7
 ['muepbio',4.62963e-7],
 ['muepsbio',2.89352e-6],
 ['muezbio',3.93519e-6],
 ['nat_co2',284.32], # where is this from? not in Hayashida (2020) table, and differs from Ziehn et al (284), Oke et al (280), Law et al (285)
 ['parbio',0.43],
 ['tscav_fe',3.17098e-8],
 ['wcaco3',6.94444e-5], # matches Ziehn 2020 but differs from Hayashida (2020) table: 10/(24*60*60) = 0.00011574
 ['wdetbio',2.77778e-4] # matches Ziehn 2020 but differs from Hayashida (2020) table: 18/(24*60*60) = 0.00020833
]
```
also I guess you already know that some of your parameters in Hayashida et al (2020) don't match Ziehn et al (2020):

![Screen Shot 2021-10-29 at Fri 29-10 12 56pm 1](https://user-images.githubusercontent.com/31054815/139360676-cb90efc5-b4ca-469c-9f35-ba802bfd88f2.png)

**Comment by hakaseh** [comment]
@aekiss thanks, it's good to get these reviewed by you.

['f_inorg',0.062] corresponds to `inorganic fraction` in Ziehn et al. (2020) (Z20 hereafter).
['kcbio',21.2766] interesting.. i never noticed this, but this parameter is not used in WOMBAT (`mom/src/mom5/ocean_csiro_bgc/csiro_bgc.F90`).
['kwbio',25] same as `kcbio`. this parameter is not used.
~['muecaco3',1.9838e-8] same as `kcbio`. this parameter also not used.~ (edit: see below)
['muedbio_sed',2.31481e-7] you are right that this does not match with Z20. I will check with Matt Chamberlain, as the original parameter I received from him is also this value (`/g/data/v45/hh0162/projects/icebgc/prep_omip2/from_matt/ESM1p5/bgc_param.aos.sep18.nc`).
['nat_co2',284.32] equals the natural CO2 of OMIP-BGC protocol (Orr et al. 2017, GMD).
[`wcaco3` and `wdetbio`] Matching with Z20 is what we want. Hayashida et al. (2020) is based on OFAM3, so is an older set of parameters.

**Comment by hakaseh** [comment]
i checked the wombat code, and found where the remineralisation rate is computed. In conclusion, Table of Z20 is correct.
`muedbio` is the remineralisation rate at <180m. In wombat, this rate is multiplied by 0.5 if depth is greater or equal to 180m: https://github.com/mom-ocean/MOM5/blob/6256fdc2a52ed47cf207f314ff5b7bdbf88162e0/src/mom5/ocean_csiro_bgc/bio_v3.inc#L306

**Comment by hakaseh** [comment]
One correction from my previous comment.
['muecaco3',1.9838e-8] is the same as remineralisation rate for caco3 in Z20 (0.001714 day-1), and this is used in wombat: https://github.com/mom-ocean/MOM5/blob/6256fdc2a52ed47cf207f314ff5b7bdbf88162e0/src/mom5/ocean_csiro_bgc/bio_v3.inc#L309

**Comment by aekiss** [comment]
Thanks @hakaseh. I think I was confused by the table in your 2020 paper which describes `muedbio_sed` as "Remineralization rate constant (≥ 180 m)".
As far as I can see this the only calculation that uses `muedbio_sed`: https://github.com/mom-ocean/MOM5/blob/6256fdc2a52ed47cf207f314ff5b7bdbf88162e0/src/mom5/ocean_csiro_bgc/csiro_bgc.F90#L662
So I think it's clear from the code that `muedbio_sed` is not "Remineralization rate constant (≥ 180 m)".

If `muedbio_sed` is actually not the >180m remin rate, where does its value come from? It's apparently not in Z20.
Our `muedbio_sed` = 2.31481e-7/sec = 0.020/day, which differs from the >180m remin rate of 0.024/day in Z20.
and isn't half the Z20 value of `muedbio` = 5.55556e-7/sec = 0.048/day.

Should `muedbio_sed` match the >180m remin rate 2.77778e-07/sec = 0.024/day? Or are they independent?

**Comment by aekiss** [comment]
Sorry @hakaseh, ignore my last post - I just read Matt's email, which I'll add here for the record.

> The value you state, 2.31481e-7 s-1, is the value that was used and frozen into the ESM1.5 version while producing output for CMIP6.  
muedbio_sed is just applied to the remineralisation of the sediments, and isn't actually discussed in Ziehn et al 2020.  
Originally, the muedbio_sed value was a copy of muedbio (remineralisation of detritus sinking through the water column); however, it looks like at some time muedbio was updated while tuning the ocean-BGC and muedbio_sed missed out.  
Going forwards, I would match muedbio and muedbio_sed if starting a new block of work, though ESM1.5 is static (the impact is negligible, this sediment pool of nutrients is primarily for model stability and not direct study).  

So in conclusion we are keeping `muedbio_sed` = 2.31481e-7/sec to be consistent with ESM1.5, right?

**Comment by hakaseh** [comment]
yes, our value is consistent with ESM1.5.
sorry for the confusion from my paper. that's definitely my misunderstanding of `muedbio_sed`.

**Comment by aekiss** [comment]
All good, that all seems fine so I'll close this issue.
